engine.go

```diff
@@ -148,7 +154,7 @@ func (eng *Engine) measure(t time.Time, name string, value interface{}, ftype Fi
 	m.Tags = append(m.Tags[:0], eng.Tags...)
 	m.Tags = append(m.Tags, tags...)
 
-	if len(tags) != 0 && !TagsAreSorted(m.Tags) {
+	if len(tags) != 0 && !eng.SkipTagDuplicateRemoval && !TagsAreSorted(m.Tags) {
 		SortTags(m.Tags)
```

Comment:
Do we still want to sort tags even if we are not removing duplicates?

Reply:
Can we rename the SortTags function to `RemoveDuplicates` that's really what it's doing, the sorting is an artifact of that.

Reply:
It makes sense but I can see other usages such as [flagon](https://github.com/segmentio/flagon/blob/2c3f3b2c2a11426b07b509d749e0e392cf0eb7af/cmd/flagon/cmd/helpers.go#L66). Do you want to rename to `RemoveDuplicates` because it doesn't offer the backwards compatibility?